src/modules/scraper/adapter/messaging/DepositFilledDateConsumer.ts

```diff
@@ -0,0 +1,71 @@
+import { OnQueueFailed, Process, Processor } from "@nestjs/bull";
+import { Logger } from "@nestjs/common";
+import { Job } from "bull";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { DateTime } from "luxon";
+import { EthProvidersService } from "../../../web3/services/EthProvidersService";
+import { DepositFilledDateQueueMessage, ScraperQueue } from ".";
+import { Deposit, DepositFillTx } from "../../model/deposit.entity";
+
+@Processor(ScraperQueue.DepositFilledDate)
+export class DepositFilledDateConsumer {
```

Comment:
This consumer computes the filled date for a deposit. The messages to this consumer are sent in 2 ways:
1. after the [consumer](https://github.com/across-protocol/scraper-api/blob/amatei/acx-426-add-filled-timestamp-to-deposits/src/modules/scraper/adapter/messaging/FillEventsConsumer.ts) which processes the Fill events is done, then it will submit another message to `DepositFilledDateConsumer`
2. Filled date can be computed retroactively for older deposits by calling `/scraper/deposit-filled-date` endpoint, which sends the messages to `DepositFilledDateConsumer`